recipes/_deps_ubuntu_16.04.rb

```diff
@@ -0,0 +1,46 @@
+#
+# Cookbook Name:: mcrouter
+# Recipe:: _deps_ubuntu-16.04.rb
+#
+# Copyright 2015 EverTrue, Inc.
+# Copyright 2017 Roblox, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+include_recipe 'apt'
```

Comment:
I don't think you need the `apt` cookbook or recipe any more. The `apt_update` resource is included in Chef by default in versions >= 12.7.